### ISSUE TITLE
Markup not escaped properly

### ISSUE THREAD

**Issue by wezm** [wezm/read-rust#48]
The title of this post does not get escaped properly even though content in `{{ liquid }}` tags is meant to be escaped by default:

https://github.com/wezm/read-rust/blob/1ee1b690cacb41c600aada70fa2ce91c13d3f4a8/content/_data/rust/posts.json#L231

**Comment by wezm** [comment]
Cobalt 0.15 seems to have fixed the title. The body still has issues (the `Rc<Vec<u32>>` is not escaped):

```
<blockquote>This post examines a particular, seemingly simple problem: given ownership of a Rc<Vec<u32>>, can we write a function that returns an impl Iterator<Item = u32>? It turns out that this is a bit harder than it might at first appear – and, as we’ll see, for good reason. I’ll dig into what’s going on, how you can fix it, and how we might extend the language in the future to try and get past this challenge.</blockquote>
```

**Comment by wezm** [comment]
Fixed by #112
